SkyrimNet-Zonos.py

```
@@ -47,12 +49,13 @@
 models_dict, models_values = config.load_configuration()
 AI_MODEL_DIR_TF, AI_MODEL_DIR_HY = config.get_model_paths()
 disable_torch_compile_default = config.get_disable_torch_compile_default()
-
+IGNORE_PING = True
+get_embed_cache_dir
```

Ruff (0.14.1)

Found useless expression. Either assign it to a variable or remove it.

(B018)